projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss

```diff
@@ -1,7 +1,8 @@
 :host {
   display: flex;
   flex-direction: column;
-  height: 100%;
+  padding-left: 1.5rem;
```

Comment:
Padding will not work as the `hr` elements need to span from the entire sidebar. That is the purpose of the `& > :not(hr)` selector

Reply:
Update: `hr` are not supposed to span entire sidebar